1.0/docs/devguide/registering-elements.md

```diff
@@ -232,56 +308,104 @@ local DOM when the element is constructed.
 
+Note that while the life cycle callbacks listed above will be called in the
+described order for any given element, the  **initialization timing between
+elements may vary** depending on many factors, including whether or not the
+browser includes native support for web components.
 
-### Initialization order {#initialization-order}
+#### Initialization timing for light DOM children
 
-The element's basic initialization order is:
+There are no guarantees about the initialization timing of light
+DOM children. In general elements are initialized in document order,
+so children are usually initialized after their parents.
 
-- `created` callback  
-- local DOM initialized 
-- `ready` callback
-- [`factoryImpl` callback](#custom-constructor)
-- `attached` callback
+For example, consider this light DOM for an element `avatar-list`:
 
-Note that the **initialization order may vary** depending on whether or not the
-browser includes native support for web components. In particular, there are no
-guarantees with regard to initialization timing between **sibling elements** or
-between **parents and light DOM children**. You should not rely on observed
-timing to be identical across browsers, except as noted below.
+    <avatar-list>
+      <my-photo class="photo" src="one.jpg">First photo</my-photo>
+      <my-photo class="photo" src="two.jpg">Second photo</my-photo>
+    </avatar-list>
 
-For a given element:
+`<avatar-list>` is _likely_ to have its `ready` method called before the various
+`<my-photo>` elements do.
 
-*   The `created` callback is always called before `ready`.
-*   The `ready` callback is always called before `attached`.
-*   The `ready` callback is called on any **local DOM children** before it's called
-    on the host element.
+In addition, the user can add light children at any time after
+the parent element has been created. A well-designed element
+should handle having its light DOM manipulated at runtime.
 
-This means that an element's **light DOM children** may be initialized **before or after** 
-the parent element, and an element's **siblings may become `ready` in any order**.
+To avoid timing issues, you can use the following strategies:
 
-For accessing sibling elements when an element initializes, you can call `async` from inside
-the `attached` callback:
+*   Handle light DOM children lazily. For example, a popup menu
+    element may need to count its light DOM children. By counting
+    its `children` when the menu is opened, it can handle the user
+    adding and removing menu items with minimal overhead.
+
+*   To react when children are added and removed, use the
+    [`observeNodes` method](local-dom.html#observe-nodes).
+
+#### Initialization timing for local DOM children
+
+In terms of local DOM and initialization timing, local DOM children are created,
+their property values are set as specified in the template, and `ready` is
+called on them _before_ their parent's `ready` callback is called.
```

Comment:
Haha, this is the opposite of what I expected, but makes more sense after thinking about it. I'm pretty sure I ended up relying on this behavior for a bug fix or two but still forgot.